examples/magento-graphcms/pages/search/index.tsx

```diff
@@ -14,20 +11,22 @@ import {
   ProductFiltersDocument,
   ProductListQuery,
   ProductFiltersQuery,
-  ProductFiltersProAllFiltersChip,
-  ProductFiltersProLimitChip,
   AddProductsToCartForm,
+  FilterLayout,
+  ProductListCount,
+  ProductListFilters,
+  ProductListItems,
+  ProductListPagination,
+  ProductListSort,
```

Comment:
I don't this is correct. The ProductListCount**Search** is by design, so it can be overwritten specifically for the search page.